src/app/widgets/Visualizer/PrimaryVisualizer.jsx

```diff
@@ -77,15 +79,15 @@ const PrimaryVisualizer = ({ actions, state, capable, showLoading, showRendering
                             state={state}
                             actions={actions}
                         />
-                        <Visualizer
+                        <VisualizerWrapper
                             show={showVisualizer}
                             cameraPosition={cameraPosition}
-                            ref={visualizerRef}
+                            ref={visualizer}
```

Comment:
Can just pass the visualizerRef prop here instead ( ex. **ref={visualizerRef}** )